figures/backfill.py

```diff
@@ -67,7 +67,7 @@ def backfill_enrollment_data_for_site(site):
             enrollment_data.append((obj, created))
         except CourseNotFound:
             errors.append('CourseNotFound for course "{}". '
-                          ' CourseEnrollment ID='.format(str(rec.course_id,
-                                                         rec.id)))
+                          ' CourseEnrollment ID='.format(str(rec.course_id),
+                                                         rec.id))
```

Comment:
An additional `{}` is needed. I've added a suggestion to fix the issue.

```suggestion
	            errors.append(
	                'CourseNotFound for course "{course}". CourseEnrollment ID={rec}'.format(
	                    course=str(rec.course_id),
	                    rec=rec.id,
	                )
	            )
```